### PR TITLE
admin login/logout added to the project

### PR DESCRIPTION
**Pull request by AMCernas**
# Description

What did you implemented/Why did you implemented this:

We implemented the admin login and logout using the gem devise

## Screenshots

image

image

### CODE REVIEW

File: test/models/user_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (dagp-ucol) [inline test/models/user_test.rb:6]: Remember to delete al unused code 😯.

In the other hand LGTM, just be careful with that. 
Next time I will request changes.

**Review by dagp-ucol — APPROVED** [review]

**Review by GustavoHiguera — APPROVED** [review]
LGTM :) well done.

**Review by Reiyirak — APPROVED** [review]
LGTM

**Review by MarcoRodriguez23 — APPROVED** [review]
LGTM

File: test/models/user_test.rb (hunk 1 above)
Comment (AMCernas) [reply test/models/user_test.rb:6]: ok, thx.

Comment (AMCernas) [reply test/models/user_test.rb:6]: we will fix it.

**Pull request merged**